Upgrade to Python 3

darrenldl opened the pull request:
Along with some minor addition of spacing for code readability

darrenldl committed:
Upgraded to Python 3

darrenldl committed:
Updated README

darrenldl committed:
Added tomorrow awareness

rebeccamorgan commented:
Thanks for your contribution - Python 3 compatibility is great, but I'd like to still support Python 2. Would you be happy to make the necessary changes before I merge?

darrenldl committed:
Added python2/3 compatibility

darrenldl commented:
Added the necessary imports and modifications of bash script for Python 2/3 compatbility.

I added the "tomorrow" awareness to the fork forgetting it would be added to the PR as well. Personally I find it useful, let me know if you'd like me to remove it.

rebeccamorgan commented:
Thank you!
